docs/root/configuration/other_features/graphite_statsd_stat_sink.rst

```diff
@@ -0,0 +1,7 @@
+.. _config_stat_sinks_graphite_statsd:
```

Comment:
doc nit: I don't think this is an "other feature" can you move this to the observability section in whatever way makes the most sense?